src/apscheduler/datastores/sqlalchemy.py

```diff
@@ -231,6 +232,13 @@ async def _begin_transaction(
 
             yield conn
 
+    async def _create_schema(self, conn: Connection | AsyncConnection) -> None:
+        if not self.schema:
+            return
+
+        t = text(f"CREATE SCHEMA IF NOT EXISTS {self.schema};")
```

Comment:
If I am to merge this PR, this needs to use SQLAlchemy's own DDL: https://docs.sqlalchemy.org/en/20/core/ddl.html#sqlalchemy.schema.CreateSchema